sql/atxs/atxs_test.go

```diff
@@ -414,6 +414,53 @@ func TestGetIDsByEpoch(t *testing.T) {
 	require.EqualValues(t, []types.ATXID{atx4.ID()}, ids3)
 }
 
+func TestGetIDsByEpochCached(t *testing.T) {
```

Comment:
I think we need another test that checks if adding an ATX the DB after having its epoch queried before already also updates the cache for the next query.

Reply:
Not sure if I understand the sentence... But this test does check that adding an ATX updates the cache. Also, I added database query count checks to ensure no extra SQL queries are executed